### PR TITLE
Adding instructions in readme.md to use Drush commands in multi-site setup

### PR DESCRIPTION
**Pull request by davidferlay**
Using a Drush command on a specific sub-site may not be intuitive for users new to Drupal multi-sites setup although this multi-site template makes it quite easy, providing the right spell is casted

### CODE REVIEW

File: templates/drupal8-multisite/files/README.md [hunk 1]
```diff
@@ -55,13 +55,22 @@ Adding a new subsite requires the following steps.  For these steps, assume we'r
...
 Alternatively, a PHP shell script is provided that automates steps 1-4.  See [`psh-subsite-add.php`](psh-subsite-add.php).
 
+## Using Drush in multi-site
+
+- In a Drupal multi-site setup, sites ID are defined in [web/sites/sites.php](https://github.com/platformsh-templates/drupal8-multisite/blob/master/web/sites/sites.php) (created from [example.sites.php](https://git.drupalcode.org/project/drupal/-/blob/8.8.x/sites/example.sites.php))
+- By default in this multi-site template, 2 subsites are defined in [routes.yaml](https://github.com/platformsh-templates/drupal8-multisite/blob/master/.platform/routes.yaml): `first` and `second`
+
+Any Drush command can therefore be used on a specific subsite by using `--uri=`. Ex: 
+- `drush status --uri=first`
+- `drush status --uri=second`
```
Comment (Crell) [inline templates/drupal8-multisite/files/README.md:72]:
```suggestion
Any Drush command can therefore be used on a specific subsite by using `--uri=`.  For example:

* `drush status --uri=first`
* `drush status --uri=second`
```

File: templates/drupal8-multisite/files/README.md [hunk 2]
```diff
@@ -55,13 +55,22 @@ Adding a new subsite requires the following steps.  For these steps, assume we'r
...
 7. Edit the `sites/stuff/settings.php` file again and enable Redis by setting `$platformsh_enable_redis` to true.
 8. Commit and push that change.
 9. Profit.
 
 Alternatively, a PHP shell script is provided that automates steps 1-4.  See [`psh-subsite-add.php`](psh-subsite-add.php).
 
+## Using Drush in multi-site
+
+- In a Drupal multi-site setup, sites ID are defined in [web/sites/sites.php](https://github.com/platformsh-templates/drupal8-multisite/blob/master/web/sites/sites.php) (created from [example.sites.php](https://git.drupalcode.org/project/drupal/-/blob/8.8.x/sites/example.sites.php))
+- By default in this multi-site template, 2 subsites are defined in [routes.yaml](https://github.com/platformsh-templates/drupal8-multisite/blob/master/.platform/routes.yaml): `first` and `second`
```
Comment (Crell) [inline templates/drupal8-multisite/files/README.md:68]:
```suggestion
In a Drupal multi-site setup, sites ID are defined in [web/sites/sites.php](https://github.com/platformsh-templates/drupal8-multisite/blob/master/web/sites/sites.php).  By default in this multi-site template, 2 subsites are defined in [routes.yaml](https://github.com/platformsh-templates/drupal8-multisite/blob/master/.platform/routes.yaml): `first` and `second`
```

File: templates/drupal8-multisite/files/README.md (hunk 1 above)
Comment (davidferlay) [reply templates/drupal8-multisite/files/README.md]: Thanks ! Commit updated with your suggestion @Crell

File: templates/drupal8-multisite/files/README.md (hunk 2 above)
Comment (davidferlay) [reply templates/drupal8-multisite/files/README.md]: Thanks ! Commit updated with your suggestion @Crell

**Pull request merged**